docs/reference/setup/logging-config.asciidoc

```diff
@@ -195,7 +195,7 @@ particular logger to another file. However, these use cases are rare.
 In addition to regular logging, Elasticsearch allows you to enable logging
 of deprecated actions. For example this allows you to determine early, if
 you need to migrate certain functionality in the future. By default,
-deprecation logging is enabled at the WARN level, the level at which all
+deprecation logging is enabled at the DEPRECATION level, the level at which all
```

Comment:
The text implies that the log level can be changed, which isn't the case. Maybe we should just say:

> Deprecation log messages are emitted at the `DEPRECATION` level.

Reply:
@pugnascotia 
Aha! So if it cannot be changed, then the text box below showing the configuration (as editable) should also be removed?

Reply:
Ah, no, that's for configuring the *logger* (the thing that logs to e.g. disk), which would still result in the deprecation messages being filtered out. So we can leave that be.

Reply:
Thanks - understood.
I changed the PR to have your suggested text.

Reply:
Thanks @pugnascotia @robin13.

The `logger.deprecation.level = warn` example that immediately follows this sentence is still confusing to me. What is that example intended to show?

We show that a user can disable the deprecation logger by setting the log level to `error` later on the page.

Reply:
Ah - the level used to be `warn`, but was changed a while ago to `deprecation` as part of a refactoring effort (by me). The `deprecation` level is very slightly lower than `warn` (so in between `warn` and `info`), but I don't know that we want to get into that in the docs. Also, even though technically you could now disable deprecation logging by setting the level to `warn`, I think suggesting that it be set to `error` is clearer to users, since they might (understandably) conflate deprecation messages with warning messages.

Reply:
Thanks for the clarification. I pushed some related changes with 9436d46a9074a9fb449c57ae87a55adbea94bc43. I also reordered the section to make it more clear. Feel free to revert or let me know if I misunderstood anything.

If this looks good to both of your now, I'll get this merged in. Thanks again!